src/of.js

```diff
@@ -20,4 +19,4 @@ var _of = require('./internal/_of');
  *      R.of(null); //=> [null]
  *      R.of([42]); //=> [[42]]
  */
-module.exports = _curry1(_of);
+module.exports = _of;
```

Comment:
by inlined do you mean

`module.exports = require('./internal/_of');`

? It doesn't look like our build is equipped to handle that, it leaves the `require` in the output:

`var of = require('./internal/_of');`

Or something else you had in mind?